README.md

```diff
@@ -41,6 +41,31 @@ pip install "syntheseus[all]"
 
 See [here](https://microsoft.github.io/syntheseus/installation) if you prefer a more lightweight installation that only includes the parts you actually need.
 
+## Citation and usage
+
+Since the release of our package, we've been thrilled to see syntheseus be used in the following projects:
+
+| **Project** | **Usage** | **Reference(s)** |
+|:--------------|:-----|:-----------|
+|Retro-fallback search|Multi-step search|ICLR [paper](https://arxiv.org/abs/2310.09270), [code](https://github.com/AustinT/retro-fallback-iclr24)|
+|RetroGFN|Pre-packaged single-step models|arXiv [paper](https://arxiv.org/abs/2406.18739), [code](https://github.com/gmum/RetroGFN)|
+|TANGO|Single-step and multi-step|arXiv [paper](https://arxiv.org/abs/2410.11527)|
+|"Challenging complexity with simplicity" study|Multi-step search|JCIM [paper](https://pubs.acs.org/doi/10.1021/acs.jcim.4c00432)|
```

Comment:
`"Challenging complexity with simplicity" study` could also be just `SimpRetro` if we're going for using the model/algorithm names from those papers

Reply:
Also, their code is under [github.com/catalystforyou/SimpRetro](https://github.com/catalystforyou/SimpRetro)

Reply:
Good catch- changed!